rtvi-client-android/src/main/java/ai/rtvi/client/Legacy.kt

```diff
@@ -0,0 +1,11 @@
+package ai.rtvi.client
+
+
+@Deprecated("VoiceClient is renamed to RTVIClient")
```

Comment:
Cool, I think I had added a comment somewhere else if we should add these alias. 
It makes sense to keep in its own file.